### ISSUE TITLE
[CLOSED] FG admin

### ISSUE THREAD

**Issue by NikZy** [Fotogjengen/fg#21]
<a href="https://github.com/Mikkel94"><img src="https://avatars1.githubusercontent.com/u/22194628?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [Mikkel94](https://github.com/Mikkel94)**
_Monday Aug 21, 2017 at 17:44 GMT_
_Originally opened as https://github.com/Samfundet/fg/issues/16_

----

Requirements:
* Delete user
* Edit user
* Info about user
* Change password?

Notes:
Basically the same as poweruser, but with different info / don't need to view downloaded photos
